scripts/copy_handbook.js

```diff
@@ -0,0 +1,55 @@
+const fs = require('fs/promises');
+const path = require('path')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+
+async function copyFiles (src, dest) {
+    const files = await fs.readdir(src, {withFileTypes: true})
+    for (const file of files) {
+        if (!file.name.startsWith('.')) {   
+            if (file.isDirectory()) {
+                const newSrc = path.join(src,file.name)
+                const newDest = path.join(dest, file.name)
+                fs.mkdir(newDest, {recursive: true})
+                await copyFiles(newSrc, newDest)
+            } else {
+                const srcFile = path.join(src, file.name)
+                const destFile = path.join(dest, file.name)
+                if (!file.name.endsWith('.md')) {
+                    await fs.copyFile(srcFile, destFile)
+                } else {
+                const { stdout } = await exec(`git log -1 --pretty=format:%ci ${file.name}`, {
+                    cwd: src 
+                })
+                const header = '---\n' +
+                `originPath: ${path.join(src, file.name)}\n` +
+                `updated: ${stdout}\n` +
+                '---\n'
+                const content = await fs.readFile(srcFile)
+                const body = header + content.toString().replace('---\n---\n', '')
```

Comment:
@hardillb @joepavitt I don't think this works the same as the SED command.

`header + content.toString().replace('---\n---\n', '')` will find and replace sets of `---` that are next to each other in the _content_ only, but it should be finding and replacing after the header and content have been concatenated.

Reply:
This was fixed by Nick last night - thanks for spotting to Pez